apps/server/src/modules/management/controller/database-management.controller.ts

```diff
@@ -1,31 +1,27 @@
-import { Controller, Param, Post } from '@nestjs/common';
+import { Controller, Param, Post, All } from '@nestjs/common';
 import { DatabaseManagementUc } from '../uc/database-management.uc';
 
 @Controller('management/database')
 export class DatabaseManagementController {
 	constructor(private databaseManagementUc: DatabaseManagementUc) {}
 
-	@Post('seed')
+	@All('seed')
 	async importCollections(): Promise<string[]> {
-		const result = await this.databaseManagementUc.seedDatabaseCollectionsFromFileSystem();
-		return result;
+		return this.databaseManagementUc.seedDatabaseCollectionsFromFileSystem();
```

Comment:
why don't we need the await for the return?

Reply:
fancy typescript logic, happens implicitly